sklearn/preprocessing/_data.py

```diff
@@ -2059,9 +2066,10 @@ def binarize(X, *, threshold=0.0, copy=True):
         Threshold may not be less than 0 for operations on sparse matrices.
 
     copy : bool, default=True
-        Set to False to perform inplace binarization and avoid a copy
-        (if the input is already a numpy array or a scipy.sparse CSR / CSC
-        matrix and if axis is 1).
+        If False, try to avoid a copy and scale in place.
```

Comment:
I feel in this case (and maybe others) "scale" is not appropriate, so maybe stay closer to the original wording.